engine-agent/pom.xml

```diff
@@ -18,6 +18,13 @@
   <description>A wrapper around the test engine. This wrapper makes it possible to run the engine in a
     testcontainer.</description>
 
+  <licenses>
+    <license>
+      <name>Zeebe Community License v1.1</name>
+      <url>https://zeebe.io/legal/zeebe-community-license-1.1</url>
```

Comment:
:question: Not sure what the legal best practice is here. Should we link to the source or link to the copy in this repo?

Reply:
I just did what we did in Zeebe here and assumed it is the way to go.